tests/monitoring/vm_monitoring.go

```diff
@@ -511,3 +489,12 @@ func validateLastConnectionMetricValue(vmi *v1.VirtualMachineInstance, formerVal
 
 	return metricValue
 }
+
+func createRunningVM(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, runStrategy v1.VirtualMachineRunStrategy) *v1.VirtualMachine {
+	By("Create a running VirtualMachine")
+	vm := libvmi.NewVirtualMachine(vmi, libvmi.WithRunStrategy(runStrategy))
+	vm, err := virtClient.VirtualMachine(testsuite.GetTestNamespace(vm)).Create(context.Background(), vm, metav1.CreateOptions{})
+	Expect(err).ToNot(HaveOccurred())
```

Comment:
@avlitman for better assertion readability please use `ExpectWithOffset` since its a helper function